src/main/external-resources/renderers/Samsung-9series.conf

```diff
@@ -30,11 +31,11 @@ LoadingPriority = 1
 
 TranscodeVideo = MPEGTS-H264-AC3
 TranscodeAudio = WAV
-#MaxVideoWidth = 3840
-#MaxVideoHeight = 2160
+MaxVideoWidth = 3840
```

Comment:
As long as it has been tested that's ok. It would cause broken 4k playback if it hadn't been tested, so I wanted to make sure.